src/lib/components/Container/Container.tsx

```diff
@@ -0,0 +1,44 @@
+import { HTMLAttributes, ReactNode } from 'react';
+import styles from './Container.module.css';
+
+type ContainerProps = HTMLAttributes<HTMLDivElement> & {
+  maxWidth: number;
+  minWidth?: number;
+  verticalPadding?: number;
+  horizontalPadding?: number;
+  children: ReactNode;
+};
+
+const Container = (props: ContainerProps) => {
+  const {
+    maxWidth,
+    minWidth,
+    verticalPadding,
+    horizontalPadding,
+    children,
+    ...rest
+  } = props;
+
+  const containerStyles = {
+    paddingTop: `${verticalPadding}px`,
+    paddingBottom: `${verticalPadding}px`,
+    paddingLeft: `${horizontalPadding}px`,
+    paddingRight: `${horizontalPadding}px`,

```

Comment:
위에서 보니 이건 선택 값이던데, 값을 넣지 않으면 그냥 'px' 가 들어가는 걸까요? 

뭔가 사용할 때는 영향은 없겠지만 저는 이렇게 써 주고 싶은 욕구가 드는군요..ㅎㅎ

```suggestion
    paddingTop: `${verticalPadding || 0}px`,
    paddingBottom: `${verticalPadding || 0}px`,
    paddingLeft: `${horizontalPadding || 0}px`,
    paddingRight: `${horizontalPadding || 0}px`,
```

이거 뭔가 1줄로도 줄일 수 있을 것 같은데.. 힘드려나요? 이런식으로요!

```suggestion
    padding: `${verticalPadding || 0}px` `${horizontalPadding || 0}px`,
```

Reply:
오 좋은데요? 당장 이렇게 고쳐야겠어요!
감사합니다!!